extensions/lem-dashboard/default-dashboard.lisp

```diff
@@ -0,0 +1,93 @@
+      %%*===================        
+    %%%%%%%%%%%*+============       
+   %%%%%%%%%%%%%   +===========     
+  %%%%%%%%%%%%%%    -===========    
+ %%%%%%%%%%%%%#*+=-:::==========    
+%%%%%%%%%%%%=:::::::::-==-:%    *   
+%%%%%%%%%%*::::::::::::::::::==:    
+%%%%%%%%%*-+*##-:::::::::::::       
+%%%%%%%%%%%%%#-----                 
+%%%%%%%%%%%%%=:::::                 
+ %%%%%%%%%%%*:::::::                
+  %%%%%%%%%%=:::::::::              
+   %%%%%%%%%*:::::::::::::::        
+    %%%%%%%%%*::::::::::::          
+      %%%%%%%%%#-::::::             
+        %%%%%%%%%*-:::::            
+            #+=-::::::::            "))
+
+(define-command open-lem-docs () ()
+  (open-external-file "https://lem-project.github.io/usage/usage/"))
+
+(define-command open-lem-github () ()
+  (open-external-file "https://github.com/lem-project/lem"))
+
+(defun set-default-dashboard (&key 
+                              (project-count 5)
+                              (file-count 5)
+                              (splash *default-splash*)
+                              (footer-messages *default-footer-messages*)
+                              hide-links)
+  (let ((dashboard-items 
+          (list (make-instance 'dashboard-splash
+                               :item-attribute 'document-metadata-attribute
+                               :splash-texts splash)
+                (make-instance 'dashboard-working-dir)
+                (make-instance 'dashboard-recent-projects 
+                               :project-count project-count
+                               :bottom-margin 1)
+                (make-instance 'dashboard-recent-files 
+                               :file-count file-count
+                               :bottom-margin 1)
+                (make-instance 'dashboard-command
+                               :display-text " New Lisp Scratch Buffer (l)"
+                               :action-command 'lem-lisp-mode:lisp-scratch
+                               :item-attribute 'document-header2-attribute
+                               :bottom-margin 2))))
+    (unless hide-links
+      (setf dashboard-items
+            (append dashboard-items
+                    (list (make-instance 'dashboard-url 
+                                         :display-text " Getting Started (s)"
+                                         :url "https://lem-project.github.io/usage/usage/"
+                                         :item-attribute 'document-header3-attribute)
+                          (make-instance 'dashboard-url
+                                         :display-text " GitHub (g)"
+                                         :url "https://github.com/lem-project/lem"
+                                         :item-attribute 'document-header3-attribute
+                                         :bottom-margin 2))))
+      (define-key *dashboard-mode-keymap* "s" 'open-lem-docs)
+      (define-key *dashboard-mode-keymap* "g" 'open-lem-github))
```

Comment:
I just noticed this code, is there any reason why it is not defined at the top level?

Reply:
The way I had bindings previously worked a bit differently. The idea was to not apply the keybind until it's used in case the user has a dashboard that doesn't contain an item like the docs/github link. That being said, after changing the way the default dashboard is created, it doesn't work since it gets called at compile time, so further calls to `set-default-dashboard` won't remove the stale binds. 

I went ahead and moved it to the top-level, users can just replace the stale bind in their config if they want to bind something else there.